sentry_sdk/utils.py

```diff
@@ -760,20 +760,22 @@ def _get_contextvars():
         # backport (also a PyPI package) works with asyncio under Python 3.6
         #
         # Import it if available.
-        if not PY2 and sys.version_info < (3, 7):
+        if sys.version_info < (3, 7):
+            # `aiocontextvars` is absolutely required for functional
+            # contextvars on Python 3.6.
             try:
                 from aiocontextvars import ContextVar  # noqa
 
                 return True, ContextVar
             except ImportError:
                 pass
+        else:
```

Comment:
This change is necessary as a bare `contextvars` installation from PyPI on Python 3.6 is totally useless for our purposes, in those cases we do want to return `HAS_REAL_CONTEXTVARS = False`